copier/cli.py

```diff
@@ -168,6 +178,8 @@ def _worker(self, src_path: OptStr = None, dst_path: str = ".", **kwargs) -> Wor
             answers_file=self.answers_file,
             exclude=self.exclude,
             force=self.force,
+            defaults=self.defaults,
+            overwrite=self.overwrite,
```

Comment:
I'm OK about keeping the `-f` shortcut for CLI calls, but I'd prefer to remove it from API, now that we have them split.

So, this call should become:

```suggestion
            defaults=self.defaults or self.force,
            overwrite=self.overwrite or self.force,
```

Reply:
Agree, that makes sense.